src/Sylius/Behat/Page/Admin/Shipment/IndexPage.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Page\Admin\Shipment;
+
+use Sylius\Behat\Page\Admin\Crud\IndexPage as BaseIndexPage;
+
+final class IndexPage extends BaseIndexPage implements IndexPageInterface
+{
+    public function shipShipmentWithSpecificOrderNumber(string $orderNumber): void
+    {
+        $tableAccessor = $this->getTableAccessor();
+        $table = $this->getElement('table');
+        $row = $tableAccessor->getRowWithFields($table, ['number' =>$orderNumber]);
+        $field = $tableAccessor->getFieldFromRow($table, $row, 'actions');
+        $field->pressButton('Ship');
```

Comment:
This part: 

```php
$tableAccessor = $this->getTableAccessor();
$table = $this->getElement('table');
$row = $tableAccessor->getRowWithFields($table, ['number' =>$orderNumber]);
$field = $tableAccessor->getFieldFromRow($table, $row, 'actions');
```

is almost the same here and in the next function (only one parameter `actions/state`). This should be extracted and refactored a little bit (don't fear the blank lines to avoid a **wall of code** :))